frontend_lib/src/component/DisplayState/DisplayState.jsx

```diff
@@ -2,25 +2,25 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 const DisplayState = props => (
-  <div className='html-document__contentpage__textnote__state'>
-    <div className='html-document__contentpage__textnote__state__msg'>
+  <div className='displaystate'>
+    <div className='displaystate__msg'>
       <i className={`fa fa-fw fa-${props.icon}`} />
       {props.msg}
     </div>
 
-    {props.type === 'button' && (
+    {props.btnType === 'button' && (
       <button
-        className='html-document__contentpage__textnote__state__btnrestore'
+        className='displaystate__btnrestore'
```

Comment:
`btnrestore` was a specific name for when it was used only for that purpose.
Since you have now refactored it in a generic way, it can be used to display buttons for different things that restoring. So use a more generic name please